modules/logging-bucket/main.tf

```diff
@@ -21,6 +21,13 @@ resource "google_logging_project_bucket_config" "bucket" {
   retention_days = var.retention
   bucket_id      = var.id
   description    = var.description
+
+  dynamic "cmek_settings" {
+    for_each = var.kms_key_name == null ? {} : { kms_key_name = var.kms_key_name }
```

Comment:
you can also do something like

```hcl
for_each = var.kms_key_name == null ? [] : [""]
content {
  kms_key_name = var.kms_key_name
}
```

Which is easier to read

Reply:
updated